src/status_im/contexts/wallet/wallet_connect/events/session_responses.cljs

```diff
@@ -121,22 +121,23 @@
                                                (get-in db [:wallet-connect/current-request :event]))]
      (let [method      (data-store/get-request-method event)
            web3-wallet (get db :wallet-connect/web3-wallet)]
-       {:db (assoc-in db [:wallet-connect/current-request :response-sent?] true)
-        :fx [[:effects.wallet-connect/respond-session-request
-              {:web3-wallet web3-wallet
-               :topic       topic
-               :id          id
-               :result      result
-               :error       error
-               :on-error    (fn [error]
-                              (log/error "Failed to send Wallet Connect response"
-                                         {:error                error
-                                          :method               method
-                                          :event                :wallet-connect/send-response
-                                          :wallet-connect-event event}))
-               :on-success  (fn []
-                              (rf/dispatch [:wallet-connect/redirect-to-dapp])
-                              (log/info "Successfully sent Wallet Connect response to dApp"))}]]}))))
+       (when web3-wallet
```

Comment:
why is this needed? When would we send a response without the web3-wallet already initialized?

Reply:
the only reason for this issue https://github.com/status-im/status-mobile/issues/21853#issue-2751138384 when i was checking it was this event triggered before web3-wallet have initiated

Reply:
might be that it's no longer needed, since we're not supposed to process requests without logging in i.e. without web3-wallet being initialized.

Reply:
Sorry was busy with another issue, I just removed it and will do test if i can still reproduce the issue.

Reply:
removed